frontend/src/components/user/SaveSearchPanel.tsx

```diff
@@ -11,7 +9,8 @@ export const SaveSearchPanel = () => {
   const path = usePathname();
   const searchParams = useSearchParams();
   const query = searchParams?.toString() ? `?${searchParams.toString()}` : "";
-  const url = `${environment.NEXT_PUBLIC_BASE_URL}${path}${query}`;
+  const origin = typeof window !== "undefined" ? window.location.origin : "";
```

Comment:
my gut is telling me we should memoize this but it may not matter. I ended up memoizing in my branch though, fwiw https://github.com/HHS/simpler-grants-gov/pull/4073/files#diff-00a0460a1f9ea20f824eeb39f592f53c68adfa957f1d04d64959829f9700ccecR40

Reply:
Thanks for the suggestion. I spent way too much time thinking about this, and don't think the calculation of `.toString()` and grabbing the window object are worth caching, but prefer to have consistency so updated.